website/content/en/docs/getting-started/_index.md

```diff
@@ -229,6 +229,8 @@ kubectl delete node $NODE_NAME
 To avoid additional charges, remove the demo infrastructure from your AWS account.
 
 ```bash
+helm uninstall karpenter --namespace karpenter
```

Comment:
Why do we need to uninstall karpenter if we're deleting the cluster?

Reply:
I'm not 100% certain that it is necessary, but the helm chart uses the IAM Service Account and I wanted to avoid the dependency blocking the deletion of the IAM Service Account.

Reply:
Certainly doesn't hurt, just adds a few seconds.